Entities.js

```diff
@@ -0,0 +1,339 @@
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Dimensions, Keyboard } from 'react-native';
+import { React, Component, useEffect, useState, useRef } from 'react';
+import { UseOnKeyPress, UseOnKeyRelease } from "./Key.js"
+import { CreateCircle, CreateBrick, DeleteEntity } from "./Dodge.js"
+import Matter from "matter-js"
+
+const MAXwidth = Dimensions.get('window').width;
+const MAXheight = Dimensions.get('window').height;
+let counter = 0
+let GlobalX = 400
```

Comment:
You should be able to get rid of GlobalX and GlobalY and refer to this.parent.ents["knight"].x